src/main/java/com/maxmind/minfraud/request/Email.java

```diff
@@ -74,11 +95,32 @@ public Email build() {
     }
 
     /**
-     * @return The MD5 of the email address used in the transaction.
+     * @return The email address field to use in the transaction. This will be
+     * a valid email address if you used {@link #address(String)}, an MD5 hash
+     * if you used {@link #hashAddress()} as well, or null if you did not set an
+     * email address.
      */
     @JsonProperty("address")
+    public String getAddress() {
+        if (address == null) {
+            return null;
+        }
+        if (hashAddress) {
+            return DigestUtils.md5Hex(address);
+        }
+        return address;
+    }
+
+    /**
+     * @return The MD5 hash of the email address if you set one, or null if you
+     * did not.
+     */
+    @JsonIgnore
```

Comment:
Let's mark this as `@Deprecated` and add the appropriate javadoc message.

Reply:
I did this. I noticed I had broken `@link` references too, which Travis didn't catch. I wonder if we should make Travis build the Javadoc too for this reason. I'm going to try enabling that and breaking it on purpose to test it.

Reply:
Yep, it broke. I've dropped the commit. [Here's](https://travis-ci.org/maxmind/minfraud-api-java/builds/272972019) the Travis build showing it.